docs/docs/actions.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Actions"
+disableTableOfContents: true
+jsdoc: ["gatsby/src/redux/actions/public.js", "gatsby/src/redux/actions/restricted.js"]
+---
+
+import APIReference, {APIContents} from "../../www/src/components/api-reference"
+
+
+Gatsby uses [Redux](http://redux.js.org) internally to manage state. When you implement a Gatsby API, you are passed a collection of actions (equivalent to `boundActionCreators` in Redux) which you can use to manipulate state on your site.
+
+The object `actions` contains the functions and these can be individually extracted by using ES6 object destructuring.
+          
+```javascript
+// For function createNodeField
+exports.onCreateNode = ({ node, getNode, actions }) => {
+  const { createNodeField } = actions
+}
+```
+
+## Functions
+<APIContents docs={props.mergedFuncs} />
```

Comment:
I'm thinking of putting this in the template itself so we don't have to deal with passing props to the markdown.